README.md

```diff
@@ -1,2 +1,160 @@
+
+## 📗 Table of Contents
+
+- [📖 Decode Morse Code ](#decode-morse-code)
+  - [🛠 Built With ](#built-with)
+    - [Tech Stack ](#tech-stack)
+    - [Key Features ](#key-features)
+  - [Live Demo](#live-demo)
+  - [💻 Getting Started ](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#authors)
+  - [🔭 Future Features ](#future-features)
+  - [🤝 Contributing ](#contributing)
+  - [⭐️ Show your support ](#️show-your-support)
+  - [🙏 Acknowledgments ](#acknowledgments)
+  - [📝 License ](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Snapscan <a name="about-project"></a>
+
+**Snapscan** is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Authentication**
+- **Authorization**
+- **Unit and Integration Tests**
+- **Figma Design**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo](https://www.loom.com/share/251368f90b724dcaaae58936fd4fd53a?sid=b14a6a4a-e243-4d1a-bc73-fa376a101421)
+
+- [Deployment](https://snapscan-khanh.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+    ruby
+    bundler
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+  git clone https://github.com/khanh-devos/rails-7-snapscan.git 
+```
+
+### Install
+
+Install this project with:
+
+```bash
+  cd the_new_folder_name
+  bundle install
+```
+
+it will install the required gemfile for running the project
+
+### Usage
+
+```bash
+  rails s
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="author"></a>
+
+👤 **Khanh**
+
+- GitHub: [khanh-devos](https://github.com/khanh-devos)
+- Twitter: [@khanhror](https://twitter.com/home?lang=en)
+- LinkedIn: [Khanh RoR](https://www.linkedin.com/in/khanh-dom/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Initial Page**
+- **Hamburger Button**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/khanh-devos/snapscan/issues).
```

Comment:
- [**Optional**] Hi @khanh-devos, to ease contribution to our project :+1: , let's kindly make sure the  link to the issue page is working. Right now I got a 404 error page.